Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -260,7 +260,9 @@ public class Fuzzer {
     ///   - program: The program to import.
     ///   - doDropout: If true, the sample is discarded with a small probability. This can be useful to desynchronize multiple instances a bit.
     ///   - isCrash: Whether the program is a crashing sample in which case a crash event will be dispatched in any case.
-    private func internalImportProgram(_ program: Program, withDropout doDropout: Bool, isCrash: Bool) {
+    private func internalImportProgram(
+        _ program: Program, withDropout doDropout: Bool, isCrash: Bool, collectTypes: Bool = false
```

Comment:
I'm not super happy about the default value in this internal function. I think it would be better if `importProgram` required an explicit `collectRuntimeTypes: false` argument?
On the other hand, if you add some kind of `typeCollectionStatus` enum to the `Program` class (see below), you probably wouldn't need these new parameters at all.

Reply:
so we run `collectRuntimeTypes` only when current program status in `.notAttempted`?

Reply:
Yeah exactly, `if config.collectRuntimeTypes && program.typeCollectionState == .notAttempted`